Lesson7/deniz/style.css

```diff
@@ -0,0 +1,370 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+
+:root {
+    --white: #ffffff;
+    --black: #000000;
+    --grey-lighter: #555;
+    --black-darker: #242424;
+    --white-darker: #F2F2F2;
+    --green:  #9acd32;
+}
+
+
+main {
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: var(--white);
+    
+}
+
+.container-top {
+  width: 100%;
+}
+
+
+.h1-top {
+    padding-left: 2rem;
+  text-align: left;
+    color:  var(--black);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 42px;
+    margin-top: 80px;
+    margin-bottom: 40px;
+   width: 50%;
+   }
+   
+   h1 {
+    text-align: left;
+    color:  var(--black);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 42px;
+    margin-top: 80px;
+    margin-bottom: 40px;
+   }
+
+.content-one {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 50% auto;
+}
+
+
+
+  .top-div {
+    display: flex;
+    
+    margin-left: 10px;
+    gap: 10px;
+}
+
+
+.profile-info {
+    display: flex;
+    width: 50% auto;
+    flex-wrap: wrap;
+}
+
+
+
+.datum a {
+    text-decoration: none; 
+  color: black;   
+}
+
+.datum span {
+    color: var(--grey-lighter);
+}
+
+
+.folowbtn {
```

Comment:
Same here about unused class. Could it be 'fbtn' in HTML?

Reply:
yes. it supposed to be like that...